it/uk/gov/hmrc/transitmovements/repositories/DeparturesRepositorySpec.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.transitmovements.repositories
+
+import org.mongodb.scala.bson.BsonString
+import org.mongodb.scala.model.Filters
+import org.scalatest.DoNotDiscover
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
+import play.api.Logging
+import play.api.test.DefaultAwaitTimeout
+import play.api.test.FutureAwaits
+import uk.gov.hmrc.mongo.MongoComponent
+import uk.gov.hmrc.mongo.test.DefaultPlayMongoRepositorySupport
+import uk.gov.hmrc.transitmovements.models.Departure
+import uk.gov.hmrc.transitmovements.models.DepartureId
+import uk.gov.hmrc.transitmovements.models.EORINumber
+
+import java.security.SecureRandom
+import java.time.Clock
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+import scala.concurrent.ExecutionContext.Implicits.global
+
+@DoNotDiscover
```

Comment:
We will want this discovering!

Reply:
I agree, once its all working :) a few things to consider here - its on my WIP list

Reply:
Was this ever fixed? This `@DoNotDiscover` should have been fixed before this was merged.